amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -330,7 +341,7 @@ export class TableDetail extends React.Component<TableDetailProps & RouteCompone
     } else {
         innerContent = (
           <div className="container table-detail">
-            <Breadcrumb path='/' text='Search Results'/>
+            { this.getBreadcrumb() }
```

Comment:
I'm fine with this approach for now, if we merge it in as is we should file a ticket to shift to the the alternative of putting this logic in `Breadcrumb` since we'll have to use it on the `ProfilePage` as well. cc @danwom since you're working on getting the `ProfilePage` ready.